src_c/constants.c

```diff
@@ -317,6 +317,11 @@ MODINIT_DEFINE(constants)
     //Still to be decided
     DEC_CONSTS(BUTTON_WHEELUP, PGM_BUTTON_WHEELUP);
     DEC_CONSTS(BUTTON_WHEELDOWN, PGM_BUTTON_WHEELDOWN);
+    DEC_CONST(AUDIO_ALLOW_FREQUENCY_CHANGE);
+    DEC_CONST(AUDIO_ALLOW_FORMAT_CHANGE);
+    DEC_CONST(AUDIO_ALLOW_CHANNELS_CHANGE);
+    DEC_CONST(AUDIO_ALLOW_ANY_CHANGE);
```

Comment:
Do these constants need to be set to their corresponding SDL constants?

```
SDL_AUDIO_ALLOW_FREQUENCY_CHANGE
SDL_AUDIO_ALLOW_FORMAT_CHANGE
SDL_AUDIO_ALLOW_CHANNELS_CHANGE
SDL_AUDIO_ALLOW_ANY_CHANGE
```

Reply:
The macro function does that:
```
#define DEC_CONSTS_(x, y)                           \
    if (PyModule_AddIntConstant(module, x, (int)y)) \
    ADD_ERROR
#define DEC_CONST(x) DEC_CONSTS_(#x, SDL_##x)
```

Reply:
Ok, so it is being done.